walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -57,9 +60,9 @@
   /**
   * Create a List of TimeRanges that will not work for the given attendees based on events that are scheduled.
   */
-  public void createListOfBlockedTimes(Collection<Event> events, MeetingRequest request, Collection<TimeRange> blockedTimes) {
+  public void createListOfBlockedTimes(Collection<Event> events, MeetingRequest request, Collection<TimeRange> blockedTimes, Collection<String> totalAttendees) {
```

Comment:
It seems like `request` isn't used by the function anymore, please remove.

Reply:
Also, I think it'd be better for this function to return a Collection of blocked times, rather than populating one an passing it in. 

above it'd probably be better to do
`Collection<TimeRange> optionalBlockedTimes = createListOfBlockedTimes(...)`

same thing for `createListOfAvailableTimes `

Reply:
Made the change, thank you!